source/treeInterceptorHandler.py

```diff
@@ -126,6 +126,15 @@ def _set_passThrough(self, state):
 					review.setCurrentMode('document',True)
 			braille.handler.handleGainFocus(self)
 
+	def _get_isSelectionAnchoredAtStart(self):

```

Comment:
> 1. This should be in DocumentTreeInterceptor, since text is not valid for non-document TreeInterceptors.

I believe I put it in the superclass to avoid having to check for that specific subclass when using this attribute. See my previous comment. This would of course not be necessary if TextInfoRegion is always paired with a DocumentTreeInterceptor, but the documentation doesn't make that clear.